src/components/EmbeddedServices.css

```diff
@@ -0,0 +1,53 @@
+h1, h2, h3, h4, h5, h6 {
+  font-family: 'Lato', sans-serif;
+  font-weight: 300; }
+
+/*$themeColor : #112a39!important;
+$hoverColor :#fff;
+$Block: #233f53;
+$hoverBlock:#0d202c;
+$font:'Open Sans', sans-serif;
+$textColor:#fff;
+$bgColor:#112a39!important;
+$pageColor:#f7f7f7;
+$smallLine:#415d6e;//line at the intro
+$font1: 'Lato', sans-serif;
+*/
+.show-grid-fluid-embedded-services {
+  background-color: #f7f7f7; }
+
+.show-grid-embedded-services {
+  background-color: #f7f7f7; }
+
+.show-title-col {
+  background-color: #f7f7f7;
```

Comment:
Move all the color codes to a single .sass file and refer it from there

Reply:
i did it